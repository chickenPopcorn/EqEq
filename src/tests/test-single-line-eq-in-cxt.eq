SomeContext = {
  a = 42.333;
  b = 41.333;
  c = 40.333;
}

SomeContext:find a {
  print("%.0f\n", a);
  print("%.0f\n", b);
  print("%.0f\n", c);
}
