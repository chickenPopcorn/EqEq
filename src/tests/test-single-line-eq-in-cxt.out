42
41
40
